system-tests/lib/cre/types.go

```diff
@@ -679,6 +703,16 @@ func (m DonsMetadata) validate() error {
 	return nil
 }
 
+// Get the bootstrap node from the first DON that contains one. Currently only one bootstrap node is supported.
+func (m DonsMetadata) BootstrapNode() (*NodeMetadata, error) {
+	for _, don := range m.dons {
+		if don.ContainsBootstrapNode() {
+			return don.GetBootstrapNode()
```

Comment:
as mentioned above: are we fine with returning the first one we have without making sure there can ever be only one?